src/components/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+ import {useParams} from 'react-router-dom'; 
+import { useState, useEffect } from 'react';
+import {getArticles } from '../utils/api';
+import axios from 'axios';
+import '../App.css';
+
+
+const Articles = () => { 
+  const [articles, setArticles] = useState([]); 
+  
+    useEffect(() => {
+  axios.get(`https://yoris-nc-news.onrender.com/api/articles`)
+      .then(({data}) => {       
+         return setArticles(data);
```

Comment:
Don't need to return from here, can just setArticles(data)
You could separate out your api functions into a separate api.js file and then call it here